installers/go-server/release/go-server.init

```diff
@@ -54,13 +54,13 @@ check_proc() {
 
 start_go_server() {
     if [ "${CUR_USER}" != "root" ] && [ "${CUR_USER}" != "go" ]; then
-        log_failure_msg "Go Server can only be started as 'root' or 'go' user."
+        log_failure_msg "GoCD Server can only be started as 'root' or 'go' user."
         exit 4
     fi
 
     check_proc
     if [ $? -eq 0 ]; then
-        log_success_msg "Go Server already running on http://$HOSTNAME:$GO_SERVER_PORT/go"
+        log_success_msg "GoCD Server already running on http://$HOSTNAME:$GO_SERVER_PORT/go and https://$HOSTNAME:$GO_SERVER_SSL_PORT/go"
```

Comment:
An assumption of `GO_SERVER_SSL_PORT` cannot be made in the init script. You'd have to have lines similar to [this](https://github.com/gocd/gocd/blob/730ff1867576754414cc632957f344d0263bc06d/installers/go-server/release/server.sh#L77-L78), since `/etc/defaults/go-server` can possibly be empty and the `server.sh` script assumes defaults.